db/fixtures/miq_user_roles.yml

```diff
@@ -14,7 +14,6 @@
   - embedded_automation_manager
   - availability_zone
   - host_aggregate
-  - compute
```

Comment:
I don't really know why this file changed.  @kbrock can you explain?

Reply:
of note, the opposite shows that 1,051 features are identified in `miq_product_features.yml` that are not referenced in `miq_user_roles.yml` - which makes sense. just wanted to share for the record

```bash
for i in `grep ':identifier:' db/fixtures/miq_product_features.yml | awk '{print $2}' | sort -u` ; do
  grep -q -- "- ${i}$" db/fixtures/miq_user_roles.yml || echo "NOT REFERENCED: ${i}"
done
```